pages/learn/deploy/offline-updates.md

```diff
@@ -174,7 +174,7 @@ $ commit=$(balena app ${app_slug} | grep COMMIT | awk '{print $2}')
 $ balena preload ${tmpimg} \
     --app ${app_slug} \
     --commit ${commit} \
-    --pin-device-to-release
```

Comment:
If we do this for a device that has partial connectivity, then the remote device will start updating at the moment we run this command.